client/examples/local_quickstart_get.py

```diff
@@ -4,7 +4,7 @@
 
 local = ff.register_local()
 
-local_get = ff.get_local("local mode")
+local_get = ff.get_local("local-mode")
```

Comment:
you have a get here, how does it know what the name of local-mode is? Is this call necessary?